### DISCUSSION TITLE
Alternatives for Level 4 Bridge 3

### DISCUSSION THREAD

**Discussion by BarboneAllplan** [jwouellette/TPF-5_372-Unit_Test_Suite#5] [General]
There are 4 example projects included in the Level 4 testing. Bridge 3 "Steel and Concrete" appears to be more complex than the other 3 bridges combined.

Can we find an alternative bridge with similar features (e.g., steel girders and steel diaphragms)? Complicated projects will take longer to evaluate and may not necessarily improve results.

**Comment by jwouellette** [comment]
The point of Br03 is to test the complexities that do arise from such bridge types. This is an example of an overpass as part of a larger intersection, which commonly has such complexities in girders, diaphragms., and piers. The results in this case are very important, to establish that more complex geometries and compositions are possible.
